### PR TITLE
Gui image

### PR DESCRIPTION
**Pull request by b0balagurak**
make a screenshot and save it

### CODE REVIEW

**Comment by b0balagurak** [comment]
Let me know if anything needs changed

File: src/view/BeeUI.py [hunk 1]
```diff
@@ -57,7 +58,7 @@ def __init__(self):
         # Define the window itself
         self.root = tk.Tk()
         # Determine what the window will look like and what it does on close.
-        self.root.geometry("900x600")
+        self.root.geometry("1100x700")
```
Comment (nrhoopes) [inline src/view/BeeUI.py:61]: is this necessary?  It will break a few things.

File: src/view/BeeUI.py [hunk 2]
```diff
@@ -35,8 +35,9 @@
 from tkinter import PhotoImage
 from tkinter import filedialog
 from tkinter import *
-import os
-from PIL import Image, ImageTk
+from mss import mss
```
Comment (nrhoopes) [inline src/view/BeeUI.py:38]: remove mss if not used anymore

File: src/view/BeeUI.py [hunk 3]
```diff
@@ -452,6 +454,17 @@ def showStatus(self, rank, points):
     def showWrongGuess(self, str):
         self.correctLabel.configure(text=str, font=('Arial', 25))
 
+    # Public method saveScreenshot
+    def saveScreenshot(self):
+        self.x, self.y = self.root.winfo_rootx(), self.root.winfo_rooty()
+        self.w, self.h = self.root.winfo_width(), self.root.winfo_height()
+
+        screenShot = pyautogui.screenshot(region=(self.x, self.y, self.w, self.h))
+        filepath = filedialog.asksaveasfilename(defaultextension=[(".png")], initialdir=os.getcwd())
```
Comment (nrhoopes) [inline src/view/BeeUI.py:463]: filetypes argument is needed.  make it png so that we can save the image

**Review by nrhoopes — CHANGES_REQUESTED** [review]
See comments

File: src/view/BeeUI.py (hunk 2 above)
Comment (b0balagurak) [reply src/view/BeeUI.py:38]: fixed

File: src/view/BeeUI.py (hunk 3 above)
Comment (b0balagurak) [reply src/view/BeeUI.py:463]: fixed

File: src/view/BeeUI.py (hunk 1 above)
Comment (b0balagurak) [reply src/view/BeeUI.py:61]: ok, I have an issue with this. For some reason, pyautogui breaks our set size of the frame, idk why

Comment (nrhoopes) [reply src/view/BeeUI.py:61]: It appears that PyAutoGUI forces emojis and labels to be larger?  Might be a side effect we will have to deal with.  Also I may be mistaken in the breaking things, it doesn't appear to cause too many issues.

**Review by nrhoopes — CHANGES_REQUESTED** [review]
add a sleep timer somewhere that waits just a fraction of a second before taking the screenshot because I'm getting the "Save Screenshot" button still in the picture itself

File: src/view/BeeUI.py [hunk 4]
```diff
@@ -452,6 +454,17 @@ def showStatus(self, rank, points):
     def showWrongGuess(self, str):
         self.correctLabel.configure(text=str, font=('Arial', 25))
 
+    # Public method saveScreenshot
+    def saveScreenshot(self):
+        self.x, self.y = self.root.winfo_rootx(), self.root.winfo_rooty()
+        self.w, self.h = self.root.winfo_width(), self.root.winfo_height()
+
```
Comment (nrhoopes) [inline src/view/BeeUI.py:461]: Sleep timer here maybe?

File: src/view/BeeUI.py (hunk 1 above)
Comment (b0balagurak) [reply src/view/BeeUI.py:61]: It made them smaller but the issue was the frame was small and the welcome was cutoff a little bit. I did some digging and pyautogui does a api call to some dpiAware function when imported and any calls to that api after the import are blocked. Ill revert to old size and we can change things around later

Comment (nrhoopes) [reply src/view/BeeUI.py]: leave 1100x700 it's fine that way.  appears to be ok everywhere

**Review by nrhoopes — APPROVED** [review]

**Review by nrhoopes — DISMISSED** [review]
Feature results in unexpected behavior:
    - When using screenshot in windowed mode, and part of your window is below the task bar, taskbar will be included in the screenshot
    - When pulling the window below the task bar, a majority of screenshot is black
    - Notifications and anything else on screen shows up in screenshots

**Review by nrhoopes — APPROVED** [review]
Issues unable to be circumvented

**Pull request merged**